compiler/passes/normalize.cpp

```diff
@@ -1744,6 +1764,14 @@ static void change_method_into_constructor(FnSymbol* fn) {
     }
   }
 
+  if (ct->instantiationStyle == DEFINES_INITIALIZER) {
+    ArgSymbol* meme = new ArgSymbol(INTENT_BLANK, "meme", ct, NULL,
```

Comment:
Shouldn't an argument for the memory to construct into be taking in that memory with INTENT_REF ?